include/osquery/transport.h

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#pragma once
+
+namespace osquery {
+
+#ifdef __APPLE__
+const std::string kTLSCiphers =
+    "ECDH+AESGCM:DH+AESGCM:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:ECDH+3DES:"
+    "DH+3DES:RSA+AESGCM:RSA+AES:RSA+3DES:!aNULL:!MD5";
+#else
+const std::string kTLSCiphers =
```

Comment:
I don't think this is an issue anymore actually. We can use this more-restrictive default set for all platforms.